.github/workflows/github-actions.yaml

```diff
@@ -0,0 +1,49 @@
+#checkov:skip:CKV2_GHA_1:Comment
+name: DevOps-Together CI
+on:
+  push:
+    branches: [master]
```

Comment:
why only on master?
I would rather prefer to run linters on all branches and pull requests, so we can find potential problem before merging into master

Reply:
I thought it would be a problem on your part, that it starts the pipeline every time. 
I will correct it